src/main/java/de/melanx/skyblockbuilder/commands/TeamCommand.java

```diff
@@ -109,18 +109,25 @@ private static int createTeam(CommandSource source, String name, boolean join) {
             return 0;
         }
 
-        Team team = data.createTeam(name);
 
         if (join) {
             try {
                 ServerPlayerEntity player = source.asPlayer();
+                if (data.getTeamFromPlayer(player) != null) {
+                    source.sendFeedback(new StringTextComponent("You are already in a team, to create a new one you have to leave your team first!"), false);
+                    return 1;
```

Comment:
This is like an error, you should return 0 and it should be red. 
```suggestion
                    source.sendFeedback(new StringTextComponent("You are already in a team, to create a new one you have to leave your team first!").mergeStyle(TextFormatting.RED), false);
                    return 0;
```